CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
 - Limit MMDS data store size to `--http_api_max_payload_size`.
 - Cleanup all environment variables in Jailer.
 
+### Fixed
+
+- Renamed `--http_api_max_payload_size` parameter to `--http-api-max-payload-size`
```

Comment:
Can we just replace the old name in the original CHANGELOG entries since they are unreleased?

Reply:
Done